clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/vmimage/model/AzureNamedImage.groovy

```diff
@@ -0,0 +1,17 @@
+package com.netflix.spinnaker.clouddriver.azure.resources.vmimage.model
+
+/**
```

Comment:
We're typically leaving out author and related javadoc comments (guessing your ide dropped this here).